idt_config/linux/userdata.json

```diff
@@ -0,0 +1,59 @@
+{
+  "sourcePath": "<path to project root, e.g. /Users/foobar/iot-reference-nxp-rt1060>",
+  "freeRTOSTestParamConfigPath": "{{testData.sourcePath}}/examples/evkbmimxrt1060/test/include/test_param_config.h",
+  "freeRTOSTestExecutionConfigPath": "{{testData.sourcePath}}/examples/evkbmimxrt1060/test/include/test_execution_config.h",
+  "freeRTOSVersion": "202012.04-LTS",
+  "buildTool": {
+    "name": "nxp-build",
+    "version": "1.0.0",
+    "command": [
+      "{{config.idtRootPath}}/configs/build.sh {{testData.sourcePath}}"
+    ]
+  },
+  "flashTool": {
+    "name": "nxp-flash",
+    "version": "1.0.0",
+    "command": [
+      "{{config.idtRootPath}}/configs/flash.sh {{testData.sourcePath}}"
+    ],
+    "buildImageInfo": {
+      "testsImageName": "aws_iot_qual_test_signed.bin",
+      "demosImageName": "aws_iot_qual_test_signed.bin"
+    }
```

Comment:
Are these necessary? We are not passing them to the flash script so I think these can be removed

Reply:
I'm not sure, I'll give it a test run without them